BrainPortal/app/controllers/concerns/license_concerns.rb

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Why is this file created? In CBRAIN we have not yet used any `concerns`,w e've always put our controller common code in libraries in lib/. Please follow the convention. Look at all the other modules we already include in our controllers, they are all under lib/.

Reply:
removed